src/RpcServer/RpcServer.SmartContract.cs

```diff
@@ -107,6 +107,45 @@ private static Signers SignersFromJson(JArray _params)
             return ret;
         }
 
+        private JObject GetVerificationResult(UInt160 scriptHash, ContractParameter[] args)
+        {
+            var snapshot = Blockchain.Singleton.GetSnapshot();
+            var contract = snapshot.Contracts.TryGet(scriptHash);
+            var methodName = "verify";
+
+            using ApplicationEngine engine = ApplicationEngine.Create(TriggerType.Verification, new Transaction(), snapshot.Clone());
```

Comment:
I think that we don't need an empty object, because it will fault like `null` if it's needed.

Reply:
Add a parameter that allows the user to customize the Transaction?

Reply:
At least the signers, `verify` "always" check the `Signers`